portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/types/TempPhotosData.java

```diff
@@ -35,36 +35,36 @@
   private final UUID jobId;
 
   // Map of PhotoAlbums keyed by Album name.
-  @JsonProperty("photoAlbums")
-  private final Map<String, PhotoAlbum> photoAlbums;
+  @JsonProperty("tempPhotoAlbums")
+  private final Map<String, PhotoAlbum> tempPhotoAlbums;
 
   // Map of newAlbumIds keyed by the old album name.
   @JsonProperty("newAlbumIds")
   private final Map<String, String> newAlbumIds;
 
   public TempPhotosData(
       @JsonProperty("jobId") UUID jobId,
-      @JsonProperty("albums") Map<String, PhotoAlbum> photoAlbums,
+      @JsonProperty("tempPhotoAlbums") Map<String, PhotoAlbum> tempPhotoAlbums,
       @JsonProperty("newAlbumIds") Map<String, String> newAlbumIds) {
     this.jobId = jobId;
-    this.photoAlbums = photoAlbums;
+    this.tempPhotoAlbums = tempPhotoAlbums;
     this.newAlbumIds = newAlbumIds;
   }
 
   public TempPhotosData(@JsonProperty("jobId") UUID jobId) {
     this.jobId = jobId;
-    this.photoAlbums = new HashMap<>();
+    this.tempPhotoAlbums = new HashMap<>();
     this.newAlbumIds = new HashMap<>();
   }
 
   // Adds the <Key, PhotoAlbum> mapping provided
-  public void addAlbum(String key, PhotoAlbum album) {
-    photoAlbums.put(key, album);
+  public void addTempAlbumMapping(String key, PhotoAlbum album) {
```

Comment:
I guess, since we have two states for mappings, should we have two different mappings to maintain?

Reply:
Discussed offline - I've added more comments and precondition checks to the FlickrImporter to make this more obvious.